content/reference/sql/version-control/dolt-sql-functions.md

```diff
@@ -349,7 +368,100 @@ With result of single row:
 +------------+-----------------+------------+--------------+---------------+-------------+---------------+----------------+---------------+---------------+----------------+----------------+
 ```
 
-# Version Control Functions 
+## `DOLT_LOG()`
+
+The `DOLT_LOG` table function gets the commit log for all commits reachable from the
+provided revision's `HEAD` (or the current `HEAD` if no revisions provided). `DOLT_LOG()`
+works like [CLI `dolt log` command](../../cli.md#dolt-log).
+
+Note that the `DOLT_LOG()` table function currently has restrictions on how it can be used
+in queries. It does not support aliasing or joining with other tables, and argument values
+must be literal values.
+
+### Privileges
+
+`DOLT_LOG()` table function requires `SELECT` privilege for all tables.
+
+### Options
+
+```sql
+DOLT_LOG(<optional_revision>, <optional_revision>)
+```
+
+The `DOLT_LOG()` table function takes up to two optional revision arguments:
+
+- `optional_revision` — all commits reachable from the revision `HEAD`. If you'd like to
+  exclude commits from another revision (two dot log), you can use `..` between revisions
```

Comment:
It would be nice to have a couple words that explain dotdot log, or possibly also link out to an explanation. I don't think it's super commonly understood by most Git users and it's a pretty cool syntax to know about. Maybe just something like "all commits reachable by revision2, but NOT reachable by revision1" – that line is what made it concrete for me.